### PR TITLE
Issue 36

### PR DESCRIPTION
**Pull request by 22PoojaGaur**
PR for #36 

added github actions

Right now able to run `intermine_boot start local`.

### CODE REVIEW

**Comment by 22PoojaGaur** [comment]
@uosl 
The actions run can be viewed in actions tab here [action run](https://github.com/22PoojaGaur/intermine_boot)

**Comment by leoank** [comment]
Maybe the issue is with relative path. We can try using `Path(__file__).absolute()` instead of relative path to fix other tests.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  
+              
+
+    test2:
+        name: Test intermine_boot start local --build-images
+
+        runs-on: ubuntu-18.04
+        steps:
+            - uses: actions/checkout@v2
+            - uses: ./actions
```
Comment (leoank) [inline .github/workflows/main.yml]: Instead of this, it will be much simpler to install curl using a `run` command.
```
run: "apt install curl -y"
```

**Review by leoank — CHANGES_REQUESTED** [review]
I am not sure that we need the contents in the action directory. Are you using it for something specific?

**Comment by leoank** [comment]
I can confirm that using `Path(__file__).parent.parent.absolute()` solves the issue!

**Comment by leoank** [comment]
Sorry! Replace run command with `run: sudo apt-get install curl`

**Comment by 22PoojaGaur** [comment]
> I can confirm that using `Path(__file__).parent.parent.absolute()` solves the issue!

Hi @leoank how did you confirm?

It is working on local setup regardless of using `.absolute()` or not.
But it does not work on github actions run 

See the screenshot
Screenshot from 2020-08-06 14-16-29
link (https://github.com/22PoojaGaur/intermine_boot/runs/952970303?check_suite_focus=true)

**Comment by 22PoojaGaur** [comment]
> Sorry! Replace run command with `run: sudo apt-get install curl`

Interesting. I thought the user would be root in docker container. Do you know why sudo is needed and why still it does not ask for a password while running the workflow?

**Comment by leoank** [comment]
> I thought the user would be root in docker container

This is not always necessary. You can have users other than root. Github action containers are configured in this way and uses passwordless `sudo`.

**Comment by leoank** [comment]
> Hi @leoank how did you confirm?

I am using https://github.com/nektos/act to run github actions locally.
after replacing `return Path(__file__).parent.parent / 'docker-intermine-gradle'` with `return Path(__file__).parent.parent.absolute() / 'docker-intermine-gradle'`, all tests passed.

**Comment by leoank** [comment]
It is not ideal, but you can try printing the path that `_get_container_path()` is returning to debug the issue. Print the output of both the version of  `_get_container_path()`.

**Comment by leoank** [comment]
Checked you build logs, curl is already installed inside the container. You can remove the run command to install curl.

**Comment by 22PoojaGaur** [comment]
Actions ran at [actions](https://github.com/22PoojaGaur/intermine_boot/runs/966487547?check_suite_focus=true)

**Comment by heralden** [comment]
Looks like the tests run well and pass. Great job!

It seems we no longer use the `actions` dir. Can you delete this before we merge?

**Comment by 22PoojaGaur** [comment]
> Looks like the tests run well and pass. Great job!
> 
> It seems we no longer use the `actions` dir. Can you delete this before we merge?

Right. done.

**Review by heralden — APPROVED** [review]
Thanks! :rocket:

**Pull request merged**